svc-telemetry/telemetry/telemetry.go

```diff
@@ -40,7 +40,7 @@ import (
 // error will be passed back.
 func (e *ExternalInterface) GetTelemetryService() response.RPC {
 	commonResponse := response.Response{
-		OdataType:    "#TelemetryService.v1_2.TelemetryService",
+		OdataType:    "#TelemetryService.v1_3_1.TelemetryService",
```

Comment:
Please add TelemetryService to constants list here https://github.com/skhan-anjum/ODIM/blob/f8f7a9e2e7979001508abda6be8846c07f68f8d8/svc-telemetry/tlresponse/telemetryservice.go#L31

Reply:
done